src/Agent/NewRelic/Agent/Core/Transactions/NoOpTransaction.cs

```diff
@@ -179,6 +179,8 @@ public void SetCustomTransactionName(string name, TransactionNamePriority priori
 
         }
 
+        public void SetRequestMethod(string requestMethod) { }
+
```

Comment:
suggest using consistent formatting within a file... all the other methods could be converted to this concise format IMHO